src/libman/Resources/Text.resx

```diff
@@ -126,6 +126,20 @@
   <data name="CleanCommandRemarks" xml:space="preserve">
     <value>    Deletes any folders that become empty after this operation. </value>
   </data>
+  <data name="ConfigCommand_Description" xml:space="preserve">
+    <value>Get or set configuration settings</value>
```

Comment:
Super-nit: our verb tense usage is inconsistent in command descriptions. E.g. Clean command says it "delete**s**" folders, while Config (and Cache) says "Get or set", not "Get**s** or set**s**"